src/common/reader.js

```diff
@@ -862,6 +860,11 @@ class Reader {
 			this.setA11yMessage(annotationContent);
 		}
 
+		// Add page number as aria-label to provided node to improve screen reader navigation
+		let setA11yNavContent = (node, pageIndex) => {
```

Comment:
Is this function inside reader.js and not in the views, because views don't have localized strings? If so, we could introduce a `this._getString` function to the views. For now, you can just do it without localization.

Reply:
> Is this function inside reader.js and not in the views, because views don't have localized strings?

Yes, exactly. 

> If so, we could introduce a this._getString function to the views.

That would be quite useful! All a11y-related tweaks require some kind of additional strings for context, so not having to drag it all from the reader would be nice

Reply:
I removed the `setA11yNavContent` from the reader and just set the page label directly from epub view.

I then did try to just expose the `_getString` to views as a `getLocalizedString` to fetch the localized "page"  label and there were no issues there. So I pushed it too as a separate commit. Correct me if that's not the best way to do it - I can drop that last change